app/views/users/review.haml

```diff
@@ -1,4 +1,19 @@
 = javascript_pack_tag "intlTelInput"
 = stylesheet_pack_tag "intlTelInput"
 
-= render partial: "users/info_summary"
+.plain-pattern.border-bottom
+  .container
+    %p.text-center
+      PLEASE TAKE A MOMENT TO REVIEW THIS.
```

Comment:
Stylistically I'd probably prefer bold to all caps, as the latter feels a bit like shouting - but definitely not a merge blocker, we can debate that later :stuck_out_tongue_winking_eye:

Reply:
Oh, this probably gets changed in subsequent commits anyway...

Reply:
Yes it's deleted later.